src/global.css

```diff
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap');
+
+body {
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+.logo {
```

Comment:
Use CSS Modules instead. Use global styles only if necessary - for example, to style body or HTML. Looking at this file, it should only contain
```
@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap');

body {
  margin: 0;
  font-family: 'Poppins', sans-serif;
}
```

Also, personally, I would use another way of attaching Google fonts.

https://stackoverflow.com/questions/12316501/including-google-fonts-link-or-import